### PR TITLE
EE 3.5.12 FRENCH (plus minor changes)

### PR DESCRIPTION
**Pull request by ebeauchamps**

### CODE REVIEW

File: system/user/language/french/imglib_lang.php [hunk 1]
```diff
@@ -1,24 +1,43 @@
 <?php
+$lang => array(
```
Comment (kevincupp) [inline system/user/language/french/imglib_lang.php]: This should just be an `=` sign.

Comment (ebeauchamps) [reply system/user/language/french/imglib_lang.php]: Ah...sure. let me correct that tonight.

**Comment by ebeauchamps** [comment]
fixed

**Comment by kevincupp** [comment]
Thanks!!

**Pull request merged**